cmd/ci-secret-bootstrap/main.go

```diff
@@ -87,37 +96,65 @@ func parseOptions() options {
 	if err := fs.Parse(os.Args[1:]); err != nil {
 		logrus.WithError(err).Errorf("cannot parse args: %q", os.Args[1:])
 	}
+	if o.vaultAddr == "" {
+		o.vaultAddr = os.Getenv("VAULT_ADDR")
+	}
+	o.vaultToken = os.Getenv("VAULT_TOKEN")
```

Comment:
Should this be conditional too? At least the help says "defaults to the VAULT_TOKEN env var if unset"

Reply:
Oh, I see, nevermind.